app/forms/network-interface-edit.tsx

```diff
@@ -107,12 +114,19 @@ export function EditNetworkInterfaceForm({
                 submitTransitIp()
               }
             }}
+            validate={(value) => {
+              const error = validateIpNet(value)
+              if (error) return error
+
+              if (transitIps.includes(value)) return 'Transit IP already in list'
+            }}
```

Comment:
Doing this here instead of in the `handleSubmit` means we get a beautiful validation error inline on attempts to add a dupe.